src/ProgressOnderwijsUtils/ProgressOnderwijsUtils.csproj

```
@@ -1,8 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <Import Project="..\NugetPackagesCommon.props" />
   <PropertyGroup Label="Configuration">
-    <Version>56.0.0</Version>
-    <PackageReleaseNotes>Remove RadiusClient</PackageReleaseNotes>
+    <Version>57.0.0</Version>

```

Totally irrelevant, maar stel dat we semver zouden volgen is dit logischer
```suggestion
    <Version>56.1.0</Version>
```